thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -768,6 +770,7 @@ public Response replayAnomalyFunctionByFunctionId(@PathParam("functionId") long
     DateTime replayStart = ISODateTimeFormat.dateTimeParser().parseDateTime(replayStartTimeIso);
     DateTime replayEnd = ISODateTimeFormat.dateTimeParser().parseDateTime(replayEndTimeIso);
     FunctionReplayRunnable functionReplayRunnable = new FunctionReplayRunnable(detectionJobScheduler, anomalyFunctionDAO, mergedAnomalyResultDAO, rawAnomalyResultDAO, new HashMap<String, String>(), id, replayStart, replayEnd, false);
+    functionReplayRunnable.setForceBackfill(false);
```

Comment:
It is hard for me to understand how you setForceBackfill to 'false'  related to 'notified or unnotified'. Is it true that any generated anomalies is treated as 'notified label = false' by default. What the setForceBackfill really do?